scripts/build.ps1

```diff
@@ -44,12 +44,6 @@ foreach ($file in [System.IO.Directory]::EnumerateFiles("$PSScriptRoot\..\src",
 }
 
 foreach ($file in [System.IO.Directory]::EnumerateFiles("$PSScriptRoot\..\tests", "*.csproj", "AllDirectories")) {
-    if ($file -match "System.IO.Pipelines.Performance.Tests")
```

Comment:
I thought you didn't want these tests to run as part of the regular build? Why remove this?